store/dataStore.ts

```diff
@@ -30,4 +65,26 @@ export const useDataStore = create<DataStore>((set) => ({
       }),
     ),
   resetUser: () => set({ user: initialState.user }),
+  updateNFLTeams: (teamName, teamLogo) =>
```

Comment:
m: Add prop types to this function. Make it into a deconstructed params.

Reply:
Deconstructed it into its own params:

`NFLTeam: { teamName: string; teamLogo: string }`

Reply:
This is incorrect. We need to add types to the props. So what's better is that we deconstruct the props so the Interface can be shared. You'll need a return type too.

Reply:
Updated.